fileformats/CellLoader_BloodMap.h

```diff
@@ -11,9 +11,9 @@ class CellLoader_BloodMap : public CellLoader
 {
 public:
     CellLoader_BloodMap();
-    ~CellLoader_BloodMap();
+    virtual ~CellLoader_BloodMap() = default;
```

Comment:
This actually does slightly change the behavior. When marked as default like this, it inlines the destructor as if it was implicitly generated, which includes destructing all class members. Previously, the destructor was defined in the source file, making the member destruction local to the source. This can be relevant when using auto-destructing containers (like `vector` or `unique_ptr`) with a forward-declared type where the destructor is unknown to the header.

Reply:
I've experienced this before, so I know what you're talking about. If you want me to change the destructor to be defined in the .cpp file then I can do that.

Reply:
As long as it's still compiling, it's not too big of a deal. But it's something to keep in mind if you want to reduce the amount of nested includes with forward declarations.

Reply:
I think it might actually be wise to change it back, since I recall MSVC being more permissive than GCC and Clang for `default_delete` of forward-declared members (i.e., MSVC can still delete forward-declared things even when the destructor is `= default`).

Reply:
By the way, thanks for checking my code! Code reviews are fun :smiley: